web-api/src/storage.rs

```diff
@@ -136,10 +136,10 @@ pub(crate) trait DocumentProperty {
         &self,
         document_id: &DocumentId,
         property_id: &DocumentPropertyId,
-    ) -> Result<Option<()>, Error>;
+    ) -> Result<Option<Option<()>>, Error>;
```

Comment:
this is similar to the changes in [ET-4004] such that the property delete endpoint properly signals a `DocumentPropertyIdNotFound`.